lib/basedriver/commands/find.js

```diff
@@ -62,6 +68,74 @@ commands.findElementsFromElement = async function (strategy, selector, elementId
   return await this.findElOrElsWithProcessing(strategy, selector, true, elementId);
 };
 
+/**
+ * Find an element using a custom plugin specified by the customFindModule cap.
+ *
+ * @param {string} selector - the selector which the plugin will use to find
+ * elements
+ * @param {boolean} multiple - whether we want one element or multiple
+ *
+ * @returns {WebElement} - WebDriver element or list of elements
+ */
+commands.findByCustom = async function (selector, multiple) {
+  if (!this.opts.customFindModule) {
+    throw new Error("Finding an element using a pluagin is currently an " +
+                    "incubating feature. To use it you must manually install a " +
+                    "plugin module inside your Appium node_modules tree, " +
```

Comment:
should we explicitly say where to install it? npm might change this in the future though